src/ec/packages/ec-component-tag/tag.story.js

```diff
@@ -1,63 +1,62 @@
-import merge from 'deepmerge';
+/* eslint-disable no-param-reassign */
 import { storiesOf } from '@storybook/html';
-import { withKnobs, text } from '@storybook/addon-knobs';
+import { withKnobs, text, select } from '@storybook/addon-knobs';
 import { withNotes } from '@ecl-twig/storybook-addon-notes';
 import withCode from '@ecl-twig/storybook-addon-code';
-
+import { getExtraKnobs, buttonLabels } from '@ecl-twig/story-utils';
 import defaultSprite from '@ecl/ec-resources-icons/dist/sprites/icons.svg';
 import dataLink from './demo/data--link';
 import dataButton from './demo/data--button';
 import dataRemovable from './demo/data--removable';
 import tag from './ecl-tag.html.twig';
 import notes from './README.md';
 
+// Preserve the adapted specs.
+const prepareTag = data => {
+  const typeList = {};
+  typeList[data.tag.type] = data.tag.type;
```

Comment:
It all works fine, but i still don't understand this approach here for building a select with a single element. You don't need any processing, you just do this:
```
data.type = select(
    'tag.type',
    [data.tag.type],
    data.tag.type,
    buttonLabels.required
  );
```
Look that right now you are showing the right information, but it refers to the wrong piece of data, data.variant doesn't exist, is actually data.type, and doing this introduces a duplicated parameter in the notes tab.

Reply:
Thx !